src/main/java/uk/gov/cslearning/catalogue/api/PageParameters.java

```diff
@@ -31,4 +31,12 @@ public Pageable getPageRequest() {
         }
         return PageRequest.of(0, 10);
     }
+
+    @Override
+    public String toString() {
+        return "PageParameters{" +
```

Comment:
Should consider using `ToStringBuilder` or similar. Allows us to consistently control toString format across our components.